searchcore/src/vespa/searchcore/proton/attribute/attribute_initializer.cpp

```diff
@@ -198,7 +199,7 @@ AttributeInitializer::loadAttribute(const AttributeVectorSP &attr,
             attr->getBaseFileName().c_str());
         return false;
     } else {
-        attr->commit(serialNum, serialNum);
+        attr->commit(CommitParam(serialNum, true));
```

Comment:
Why are we using _forceUpdateStats=true_ here? That is not equal to the previous behaviour. The same comment applies for other changes in this file.

Reply:
Removed semantic change